### PR TITLE
Iptables cleanup and prompt for web server ip

### PR DESCRIPTION
**Pull request by luandro**
Added cleanup of iptables rules and prompt for web server ip addresses to solve #18.

### CODE REVIEW

File: files/usr/bin/captive-portal [hunk 1]
```diff
@@ -4,12 +4,32 @@
 # TODO add a parallel catch-all uhttpd service that shows the captive portal locally
 
 # Cleanup
+echo "Starting captive portal service."
+echo "You'll need to configure the web server running within the network."
+read -p "What is the webservers ipv4 address: " ANYGW4
+read -p "What is the webservers ipv6 address: " ANYGW6
 
-ANYGW4=10.5.32.1
```
Comment (aparcar) [inline files/usr/bin/captive-portal:8]: You could add a default value as fallback to the previous version

`ANYGW4="${ANYGW4:-10.5.32.1}"`

Comment (nicopace) [reply files/usr/bin/captive-portal:8]: The default would be the ip of anygw, so you can fetch it from your uci config.
Also, this does not seem to be used anywhere.

**Review by nicopace — CHANGES_REQUESTED** [review]

**Comment by luandro** [comment]
The way me and Marcos tested, and also the way I will use here in Moinho is by having a portal running on an external single-board computer. That's why I've added these inputs. In order to have the default working with UCI we'll have to solve #13 first.

**Comment by nicopace** [comment]
Agreed

**Pull request closed without merging**